src/main/java/com/chess/backend/services/PlayerService.java

```diff
@@ -19,4 +19,8 @@
 
         return players;
     }
+
+    public static int getBaseY(Player player) {
+        return player.getColor().position * 8;
```

Comment:
Do you need the color in backend?

Reply:
I think so, because white always has the first turn. What do you think?

Reply:
This will be handled in frontend.
In the order of the playerNames

Reply:
Hmmm I think it is quite helpful for the backend to know the color.